tee-worker/omni-executor/client-sdk/packages/client-sdk/src/lib/requests/__test__/account-store.request.test.ts

```diff
@@ -459,7 +459,7 @@ describe('account-store', () => {
     console.log('Step 1: create account store');
     await (async () => {
       // Request email verification code for next account store creation
-      await requestEmailVerificationCode({ email });
```

Comment:
In fact, we may not need to update any test cases in the client-sdk, because F/E team are currently only referencing apifox. If we must add, it might be better to add in ts-tests. Just a reminder, the client-sdk should not be maintained further in the future.